tests/ui/lint/unused/unused_attributes-must_use.rs

```diff
@@ -133,4 +135,5 @@ fn main() {
     }
     let s = PatternField { #[must_use]  foo: 123 }; //~ ERROR `#[must_use]` has no effect
     let PatternField { #[must_use] foo } = s; //~ ERROR `#[must_use]` has no effect
+    let _ = foo;
```

Comment:
`@run-rustfix` requires that all warnings are gone, and `foo` is unused, so I'm suppressing it here.